### PR TITLE
Closes #72 Simply PARAM and PARAMCD for Centiloid values

### LINKED ISSUE #72: Simply PARAM and PARAMCD for Centiloid values in inst/templates/ad_adapet.R and vignettes/adpet.Rmd
Rows are added when deriving Centiloid values from suvr in derive_extreme_records(). Instead of setting PARAM and PARAMCD for specific tracer and pipeline, one set of PARAM and PARAMCD should be used as Centiloid values are standardized values and can be used regardless of their origin.
Suggest to set PARAMCD = "CENTILD" or "CENTLD" or anything better. (max 8 characters),  PARAM = "Centiloid value derived from SUVR pipeline" 
See the discussion below for background discussion.
Image

### PR DESCRIPTION
**Pull request by jackychen651**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by manciniedoardo** [comment]
@jwang-lilly @jackychen651 please could you ensure this PR is completed by tomorrow CET afternoon so that we can go ahead and release the package? thanks

File: vignettes/adpet.Rmd [hunk 1]
```diff
@@ -205,88 +205,85 @@ c.  Based on the (`SUVRAFBB`) tracer, the (`AVID FBB SUVR PIPELINE`) method, and
...
+    set_values_to = exprs(
+      AVAL = compute_centiloid(
+        tracer = "18F-Florbetaben",
+        pipeline = "BERKELEY FBB SUVR PIPELINE",
+        ref_region = "Whole Cerebellum",
+        suvr = AVAL
       ),
-      keep_nas = TRUE
+      PARAMCD = "CENTLD",
+      PARAM = "Centiloid value derived from SUVR pipeline",
```
[the remark this answers is not in the document]
Comment (jwang-lilly) [reply vignettes/adpet.Rmd:232]: It is ok. I think traceability will be in NV table.

**Review by Fanny-Gautier — APPROVED** [review]
Looks ok now. Thank you for the updates.
